GeeksForGeeks/Kth_Smallest_Element/Problem.md

```diff
@@ -0,0 +1 @@
+Problem Link : https://github.com/Ankita1811/CompetitiveCode.git
```

Comment:
The problem link seems to be incorrect.
Please correct it 😅

Reply:
@Vishruth-S  I have changed the problem link. Can you please look into it